docs/proposals/resource-quota-scoping.md

```diff
@@ -0,0 +1,350 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Resource Quota - Scoping resources
+
+## Problem Description
+
+### Ability to limit compute requests and limits
+
+The existing `ResourceQuota` document constrains the total amount of compute
+resource requests.  This is useful when a cluster-admin is interested in
+controlling explicit resource guarantees often associated with higher SLA.
+The end-user creating the pod is expected to have intimate knowledge on
+their minimum required resource as well as their potential limits.
+
+There are many environments where a cluster-admin does not extend this level
+of trust to their end-user because user's often request too much resource, and
+they have trouble reasoning about what they hope to have available for their
+application versus what their application actually needs.  In these environments,
+the cluster-admin will often just expose a single value (the limit) to the end-user,
+and internally, they set the request as a factor of the limit to control for
+over-commit or based on known-historical trends.  In this environment, the
+cluster-admin is interested in exposing a quota to their end-users that maps
+to their desired limit instead of their request.
+
+### Ability to limit impact to node and promote fair-use
```

Comment:
I think having a separate quota only for BE pods only makes sense if you don't allow quota limits on BE pods, since otherwise there's really no difference between a BE pod and a non-BE pod (both have non-zero limit, can run briefly or forever, etc.). I do see later that you don't allow quota limits on BE pods, which makes what you're doing here make more sense... Maybe make some comment along these lines here...